packages/grid_client/src/modules/bridge.ts

```diff
@@ -5,31 +5,80 @@ import { ListenToMintCompletedModel, SwapToStellarModel } from "./models";
 
 class Bridge {
   client: TFClient;
+
+  /**
+   * Class representing a Bridge for interacting with TFClient's TFTBridge functionality.
```

Comment:
please apply it to all in the file
```suggestion
   * Class representing a Bridge for interacting with TFChain's TFTBridge functionality.
```